tmt/steps/provision/testcloud.py

```diff
@@ -480,9 +483,12 @@ def start(self):
         self.instance_name = self._tmt_name()
         self.instance = testcloud.instance.Instance(
             name=self.instance_name, image=self.image,
-            connection=f"qemu:///{self.connection}")
+            connection=f"qemu:///{self.connection}", desired_arch=self.arch)
         self.verbose('name', self.instance_name, 'green')
 
+        # Decide if we want to multiply timeouts when emulating an architecture
+        TIME_COEFF = TCG_TIMEOUT_COEF if not self.instance.kvm else 1
```

Comment:
The `TIME_COEFF` looks like a global constant which is modified here (confused me a bit). Let's rather use lower case instead?